### PR TITLE
US 1643841 - Loc fix incomplete string.

### PR DESCRIPTION
**Pull request by v-thpra**
[User Story 1643841](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1643841): Content Update - Localization fix: Quantum Docs - M44796 - Possible incomplete string

1. The Localization team identified this incomplete sentence and wanted it clarified.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/overview/what-is-qsharp.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/76bb5bef4ad15a7129bd6a5d3713c0b2f8174f8b/articles/overview/what-is-qsharp.md) | 84 | 47 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f9cb2aff-9e0e-4543-baf8-28f235842570_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/overview/what-is-qsharp.md [hunk 1]
```diff
@@ -20,12 +20,11 @@ The Q# programming language provides you with an intuitive set of types, operati
...
 
-
 ## How does Q# work?
 
 One of the fundamental building blocks of Q# is the `Qubit` type, which cannot be copied or directly accessed, just like a real qubit. Instead, we can measure it and store the outcome of the measurement in a `Result` variable, a Q# type that can take two possible values: `Zero` and `One`. Constructs like this one guarantee that algorithms always respect the laws of quantum physics and can run correctly on quantum computers or simulators.
 
-Q# also includes classical logic features like conditionals or loops with some subtleties to make sure that all the quantum rules are being respected. For example, constraining the way loops are executed to ensure quantum operations are.
+Q# also includes classical logic features like conditionals and loops with some subtleties to make sure that all the quantum rules are being respected. For example, constraining the way loops are executed to ensure that quantum operations are not called within functions which may only contain classical subroutines.
```
Comment (cgranade) [inline articles/overview/what-is-qsharp.md]:
```suggestion
Q# also includes classical logic features like conditionals and loops with some subtleties to make sure that all the quantum rules are being respected. For example, constraining the way loops are executed to ensure that quantum operations are not called within functions which may only contain deterministic classical subroutines.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/overview/what-is-qsharp.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/1efec30ee7318c2da6bc9fa67e22e51b226f0947/articles/overview/what-is-qsharp.md) | 84 | 47 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3e75927c-ba7e-4d05-95ac-c0cdd0aa1d86_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by v-thpra** [comment]
@cgranade Thanks! that's an excellent change.

**Comment by v-thpra** [comment]
@cgranade If there are no more suggested changes for this, the Localization team would like to get this merged. Thanks!

**Comment by v-thpra** [comment]
#sign-off

**Comment by v-thpra** [comment]
@bromeg If there are no more suggested changes, the Localization team would like to get this PR merged. Thanks!

**Pull request merged**